src/ui/components/pages/MyServices/MyServicesCards/MyServicesCards.tsx

```diff
@@ -44,6 +46,7 @@ export type Props = {
         | undefined;
     catalogExplorerLink: Link;
     onRequestDelete(params: { serviceId: string }): void;
+    onRequestUpgrade(serviceName: string): void;
```

Comment:
!!!!!!!   
Please try to enforce consistency.
Just look at the above line.  
If you want to rename `serviceId` to `serviceName` fine. But you do it in a separate PR and you propagate the change everywhere.